ctapipe/image/muon/fitting.py

```diff
@@ -83,6 +87,16 @@ def _psf_neg_log_likelihood(params, x, y, weights):
         (np.log(sigma) + 0.5 * ((pixel_distance - radius) / sigma)**2) * weights
     )
 
+def strip_unit_savely(x, y):
```

Comment:
This converts to si base units and returns the values in those units. I don't think we want this, right?

How about:

```
x = u.Quantitity(x, copy=False)
x, unit = v.value, x.unit
y = u.Quantity(y, copy=False).to_value(unit)

return x, y, unit
```
This will 
* not copy the data
* keep the unit of x
* make sure y is convertible to x
return the values of x y and the unit
```

Reply:
nice, thanks

Reply:
Also *safely*

Reply:
How do you like that?
I added arbitrary number of args, since for the hough fit I've already seen we need this function with 3 arguments

Reply:
is the name okay? it does not strip ... is rather splits ... hmm

Reply:
Also thought about using `*args` 

Something like: `all_to_value(*args, unit=unit)`?

And then you can do stuff like `all_to_value(x, y, r, unit=x.unit)` or `all_to_value(x, y, r, unit=u.m)`?

Reply:
hmm thats more explicit. ... so yes.

Reply:
should it still return the unit then? I guess no.

Reply:
Don't think it should.

Reply:
done .. and I assume resolved.